Makefile

```diff
@@ -67,7 +67,9 @@ watch:
 .PHONY: watch
 
 ## unit-test: Run the unit tests.
-unit-test: build/test/background.js build/test/ui.js build/test/shared-utils.js
+ESBUILD_TESTS = unit-test/background/*.js unit-test/background/**/*.js unit-test/ui/**/*.js unit-test/shared-utils/*.js
```

Comment:
(Random line, GitHub isn't great for reviews.) Can you try adjusting the Make command above to remove `j4`? That causes problems, but was useful with Browserify due to speed. I'm thinking ESBuild might be fast enough we can just remove that.

Reply:
Sure. Using -j4 does reduce the time for a clean build from 5s to 3s, but maybe that's fast enough?